dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -165,13 +168,14 @@ public Response deletePushHistory(@Context HttpServletRequest request, @PathPara
 	@GET
 	@Path("/deleteenvironmentpushhistory/{params:.*}")
 	@Produces("application/json")
-	public Response deleteEnvironmentPushHistory(@Context HttpServletRequest request, @PathParam("params") String params) {
+	public Response deleteEnvironmentPushHistory(@Context HttpServletRequest request, @PathParam("params") String params) throws JSONException, LanguageException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 160 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)